net/iputils/Makefile

```diff
@@ -0,0 +1,151 @@
+#
+# Copyright (C) 2019 Noah Meyerhans <frodo@morgul.net>
+#
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=iputils
+PKG_VERSION:=20190709
+PKG_RELEASE:=1
+
+PKG_MAINTAINER:=Noah Meyerhans <frodo@morgul.net>
+PKG_LICENSE:=BSD-3
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_SOURCE:=s$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/iputils/iputils/archive
+PKG_HASH:=a15720dd741d7538dd2645f9f516d193636ae4300ff7dbc8bfca757bf166490a
+
+PKG_INSTALL:=
+PKG_USE_MIPS16:=0
```

Comment:
I'm curious why.

Reply:
Overzealous copy & paste. It is not relevant here.